js/main.js

```diff
@@ -1,6 +1,25 @@
 var $photoUrlInput = document.querySelector('#photo-url');
 var $previewImage = document.querySelector('.preview-image');
 var $form = document.querySelector('.entry-form');
+var $tab = document.querySelector('.tab');
+var $view = document.querySelectorAll('.view');
+var $newButton = document.querySelector('.new-button');
+
+$newButton.addEventListener('click', swapHome);
+
+$tab.addEventListener('click', swapEntries);
+
+function swapEntries(event) {
+  $view[0].className = 'view hidden';
+  $view[1].className = 'view';
+  data.view = 'entries';
+}
+
+function swapHome(event) {
+  $view[0].className = 'view';
+  $view[1].className = 'view hidden';
```

Comment:
What if the HTML was reorganized? You should not operate based on index values, that leaves the door wide open for bugs